.envrctemplate

```diff
@@ -1,17 +1,21 @@
-export NODE_ENV=[node env]
-export APP_PORT=[port]
-export MONGO_ADDRESS=[db uri]
+# create states folder
+mkdir ./states
 
 # check if python version is set in current dir
```

Comment:
i've never seen this before, curious, is this to ensure that the correct python installation is being used when working?

Reply:
Yep, also automatically loads and unloads a virtual environment using direnv